mixer/pkg/config/crd/admit.go

```diff
@@ -308,6 +309,7 @@ func (ac *AdmissionController) ServeHTTP(w http.ResponseWriter, r *http.Request)
 			glog.V(4).Infof("Failed to read request body: %v", err)
 		}
 	}
+	glog.V(4).Infof("request body: %s", body)
```

Comment:
why requestBody at V(4) and message on line 302 at V(7) ?

Reply:
The request body contains the whole data for the validation (i.e. kind, name, namespace, spec, etc.) while I found that the request object itself (in line 302) doesn't show up very helpful information, like request path or http method, and no content information on request body.
I'm fine to remove the line 302.